core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.kt

```diff
@@ -396,12 +398,23 @@ abstract class CoreMainActivity : BaseActivity(), WebViewProvider {
     lifecycleScope.launch {
       externalLinkOpener.openExternalUrl(
```

Comment:
This method calls from `CoreReaderFragment`, which also opens our website, so we should show this new dialog there also.